config.json

```diff
@@ -504,6 +510,12 @@
             "CreateTestOverlay": "default"
           }
         },
+        "testHost": {
+          "description": "Runs test host target.",
+          "settings": {
```

Comment:
Is there a better description that could be used here? This means nothing to me.